client/cypress/integration/missions.spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Should test missions', () => {
+  it('Should add, edit, delete a mission', () => {
+    cy.visit('http://localhost:4200/login');
+    cy.get('[data-cy=login]').type('jenaye');
+    cy.get('[data-cy=password]').type('jenaye');
+    cy.get('[data-cy=submit]').click();
```

Comment:
Use before to connect and store token in the storage to stop re-login user for each steps